lib/iris/tests/unit/fileformats/pyke_rules/compiled_krb/fc_rules_cf_fc/test_build_cube_metadata.py

```diff
@@ -35,33 +35,33 @@
 from iris.tests import mock
 
 
-class TestInvalidGlobalAttributes(tests.IrisTest):
-    @staticmethod
-    def _make_engine(global_attributes=None):
-        if global_attributes is None:
-            global_attributes = {}
+def _make_engine(global_attributes=None, standard_name=None, long_name=None):
+    if global_attributes is None:
+        global_attributes = {}
+
+    cf_group = mock.Mock(global_attributes=global_attributes)
 
-        cf_group = mock.Mock(global_attributes=global_attributes)
+    cf_var = mock.Mock(
+        cf_name='wibble',
+        standard_name=standard_name,
+        long_name=long_name,
+        units='m',
+        dtype=np.float64,
+        cell_methods=None,
+        cf_group=cf_group)
 
-        cf_var = mock.Mock(
-            cf_name='wibble',
-            standard_name=None,
-            long_name=None,
-            units='m',
-            dtype=np.float64,
-            cell_methods=None,
-            cf_group=cf_group)
+    engine = mock.Mock(
+        cube=Cube([23]),
+        cf_var=cf_var)
 
-        engine = mock.Mock(
-            cube=Cube([23]),
-            cf_var=cf_var)
+    return engine
```

Comment:
Wondered why you pulled this out of `TestInvalidGlobalAttributes`, but you're also using it in the new test class `TestCubeName`... nice :+1: